bowler/tests/smoke.py

```diff
@@ -5,17 +5,25 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+import io
 import logging
 from pathlib import Path
 from unittest import TestCase
+from unittest.mock import Mock
 
 from fissix.fixer_util import Call, Name
 
 from ..query import Query
-from ..types import TOKEN
+from ..types import TOKEN, BadTransform
+
+STDERR = io.StringIO()
```

Comment:
Does this need the module level var, or are you just trying to silence them?

Reply:
The reason I put that at module level was for tests and/or debugging to be able to look at contents of stderr and make sure correct things were logged, while also silencing it on the console.